src/packet_analysis/Analyzer.h

```diff
@@ -68,18 +68,29 @@ class Analyzer {
 	 * protocol, the data reference should be updated to point to that payload.
 	 *
 	 * @param packet The packet to analyze.
-	 *
 	 * @param data Reference to the payload pointer into the raw packet.
 	 *
 	 * @return A tuple of analysis result and identifier. The result indicates
 	 * how to proceed. If analysis can continue, the identifier determines the
 	 * encapsulated protocol.
 	 */
-	virtual AnalysisResultTuple Analyze(Packet* packet, const uint8_t*& data) = 0;
+	virtual AnalyzerResult Analyze(Packet* packet, const uint8_t*& data) = 0;
```

Comment:
Definitely a fan of this change. Being able to just return a single value instead of a tuple from every call to this is very nice.